docs/installation/server-config/index.rst

```diff
@@ -93,12 +95,9 @@ bootstrap.servers
 ^^^^^^^^^^^^^^^^^
 
 A comma-separated list of host and port pairs that is used for connecting with a Kafka cluster. This list should be
-in the form ``host1:port1,host2:port2,...`` The default value in KSQL is ``localhost:9092``. For example, to change it to ``9095``
-by using the KSQL command line:
-
-.. code:: bash
+in the form ``host1:port1,host2:port2,...``. The default value in KSQL is ``localhost:9092``.
 
-    ksql> SET 'bootstrap.servers'='localhost:9095';
+.. important:: ``bootstrap.servers`` must not be set in the KSQL CLI via ``SET``. Define ``bootstrap.servers`` only in the KSQL server configuration file.
 
 For more information, see :ref:`Streams parameter reference <streams_developer-guide_required-configs>` and the :cp-javadoc:`Javadoc|clients/javadocs/org/apache/kafka/clients/consumer/ConsumerConfig.html#BOOTSTRAP_SERVERS_CONFIG`.
```

Comment:
should this sentence be a part of `.. important`?

Reply:
No